tests/unit/test_loaders.py

```diff
@@ -128,6 +129,23 @@ def load_file(self, name):
         loaded = loader.load_data('baz')
         self.assertEqual(loaded, ['loaded data'])
 
+    @mock.patch('os.path.isdir', mock.Mock(return_value=True))
```

Comment:
i don't think you need to create a Mock object here. can just supply the return value as a paremeter.

```suggestion
    @mock.patch('os.path.isdir', return_value=True)
```

Reply:
Making that change results in this test failure:

```
TypeError: TestLoader.test_load_data_optionally_returns_path() takes 1 positional argument but 2 were given
```

Reply:
whoops never mind then 😅